package.json

```diff
@@ -2,12 +2,15 @@
   "name": "late_term_assignment",
   "version": "1.0.0",
   "description": "Tic-Tac-Toe built using continuous delivery",
-  "main": "index.js",
+  "private": true,
   "scripts": {
     "test": "jest",
     "run": "node index.js",
```

Comment:
This is duplicate code, due to the fact that there now exists a: "nmp start" script.